src/config/remote_generic_dependency.go

```diff
@@ -12,24 +10,18 @@ type remoteGenericDependency struct {
 
 // HasDockerConfig returns a boolean indicating if a docker-compose.yml entry should be generated for the dependency
 func (g *remoteGenericDependency) HasDockerConfig() bool {
-	return true
+	return false
 }
 
 // GetDockerConfig returns docker configuration and an error if any
 func (g *remoteGenericDependency) GetDockerConfig() (types.DockerConfig, error) {
-	return types.DockerConfig{
-		Image: fmt.Sprintf("%s:%s", g.config.Name, g.config.Version),
-	}, nil
-}
-
-func (g *remoteGenericDependency) GetServiceName() string {
-	return g.config.Name + g.config.Version
+	return types.DockerConfig{}, nil
```

Comment:
We aren't using generic remote dependencies. When we do support them, we have to define a `docker-image` field somewhere under `remote.dependencies`, and specify that remote generic dependencies are for docker type dependencies only

Reply:
Lets just get rid of remote generic dependencies then and if the type isn't exocom/rds we throw an error

Reply:
Hmm, we'd be breaking a lot of apis if we started returning errors with `NewRemoteAppDependency`. Do you think it makes sense to keep remote generic dependency as a placeholder until we define what it does exactly?

Reply:
Okay. Good to leave as is for now since this will be changing soon